README.md

```diff
@@ -394,6 +394,14 @@ yarn workspace @experimenter/nimbus-ui test:cov
 
 For a full reference of all the common commands that can be run inside the container, refer to [this section of the Makefile](https://github.com/mozilla/experimenter/blob/main/Makefile#L16-L38)
 
+#### Intergration Tests
+
+Note: You need the following firefox version flag when running integration tests
+
+```sh
+FIREFOX_VERSION=nimbus-firefox-release
+```
```

Comment:
I think you could add this under https://github.com/mozilla/experimenter#integration-test-options. What do you think?

Reply:
Ya I can add it under there, but I thought we should prompt whoever is running integration tests about this before they start the test so they won't run in any errors and then look for answers. should I change it or leave it as it is? @eliserichards @yashika-khurana

Reply:
It is already noted in [some of the other commands](https://github.com/mozilla/experimenter#make-firefox_version-integration_test_nimbus), so I think adding it to the "Integration test options" section is the best way :) You can also add an example to make it more clear, like this: 

```
make FIREFOX_VERSION=nimbus-firefox-release integration_test_nimbus PYTEST_ARGS=-ktest_rollout_create_and_update
```